man/docker.1.md

```diff
@@ -26,125 +30,20 @@ To see the man page for a command run **man docker <command>**.
 **--help**
   Print usage statement
 
-**--api-cors-header**=""
-  Set CORS headers in the remote API. Default is cors disabled. Give urls like "http://foo, http://bar, ...". Give "*" to allow all.
-
-**-b**, **--bridge**=""
-  Attach containers to a pre\-existing network bridge; use 'none' to disable container networking
-
-**--bip**=""
-  Use the provided CIDR notation address for the dynamically created bridge (docker0); Mutually exclusive of \-b
-
 **--config**=""
   Specifies the location of the Docker client configuration files. The default is '~/.docker'.
 
 **-D**, **--debug**=*true*|*false*
   Enable debug mode. Default is false.
 
```

Comment:
@thaJeztah does this help ?
https://github.com/docker/docker/pull/16063#discussion_r38711266